frontend/src/components/organisms/Table/Table.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Header from '../../atoms/Header/Header';
+import TableRow from '../../molecules/TableRow/TableRow';
+import TableHeader from '../../molecules/TableHeader/TableHeader';
+import TableCell from '../../atoms/TableCell/TableCell';
+
+const Wrapper = styled.div`
```

Comment:
Whole Table should use shadow.
Please use: 
![image](https://user-images.githubusercontent.com/33421875/117198290-25514380-ade9-11eb-90a4-a2e85b46b6f7.png)

Reply:
done